core/src/test/java/org/infinispan/atomic/BaseAtomicMapLockingTest.java

```diff
@@ -174,9 +167,12 @@ public Boolean call() throws Exception {
          } else {
             blockedPrepareResponses.receive();
          }
+
+         rpcManager.expectCommand(TxCompletionNotificationCommand.class).sendWithoutResponses();
          Assert.assertTrue(txOutcome.get());
       } finally {
-         rpcManagers[txExecutor].stopBlocking();
+         rpcManager.stopBlocking();
+         rpcManager.revertRpcManager(cache(txExecutor));
```

Comment:
Either one should be enough. 

Actually could you change `revertRpcManager()` to call `stopBlocking()` first, and then `stopBlocking()` to call `assertEquals(emptyList(), queuedRequests)` instead of logging an error message (which doesn't even get printed on the console)?